examples/geojson/ai-ecosystem-example.json

```
@@ -0,0 +1,1121 @@
+{
+    "fiboa": {
+        "fiboa_version": "0.2.0",
+        "fiboa_extensions": [
+          "https://fiboa.github.io/ai-ecosystem-extension/v0.1.0/schema.yaml"
+        ]
+    },
+    "type": "FeatureCollection",
+    "crs": {
+        "type": "name",
+        "properties": {
+            "name": "urn:ogc:def:crs:OGC:1.3:CRS84"
+        }
+    },
+    "features": [
+        {
+            "type": "Feature",
+            "properties": {
+                "area": 0.011851919813431496,
+                "perimeter": 49.937117861428113,
+                "determination_method": "administrative",
+                "determination_datetime": "2022-11-30T00:00:00Z",
+                "ml:boundary_type": "field",
+                "ml:creation_date": "2024-04-25T15:57:22.015",
```

```suggestion
                "ml:creation_date": "2024-04-25T15:57:22.015Z",
```

According to the spec all date-times need to be UTC.

Also applies to all other Features.

---

This must also be fixed for all other Features, of course.